mods/common/hotkeys/observer.yaml

```diff
@@ -46,10 +46,14 @@ StatisticsCombat: F6
 	Description: Combat statistics
 	Types: Observer, Replay, Spectator
 
-StatisticsGraph: F7
+StatisticsArmy: F7
+	Description: Army statistics
+	Types: Observer, Replay, Spectator
+
+StatisticsGraph:
 	Description: Statistics graph
 	Types: Observer, Replay, Spectator
 
-StatisticsArmyGraph: F8
+StatisticsArmyGraph:
```

Comment:
... and this to `F9`. Most keyboards have keys up to `F12` or am I missing something else here?

Reply:
These keys are already bound to the replay speed controls...

Reply:
`[15:05:11] <abcdefg30> well, I guess I'll have to live with that, although I think using F9-12 for that is a waste`